server/public/model/config.go

```diff
@@ -746,7 +745,7 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 	if s.GiphySdkKey == nil {
 		switch GetServiceEnvironment() {
 		case ServiceEnvironmentProduction:
-			s.GiphySdkKey = NewString(ServiceSettingsDefaultGiphySdkKey)
+			s.GiphySdkKey = &GiphySdkKey
```

Comment:
This works but it will store in the config so the key would be accessible to admins. Should we leave the config in prod just for users to add their own key once we reintroduce that and we just override at the time of returning the config the default if none is set?

Reply:
Just trying to get some clarity on this: do we want admins to be able to set their own keys? Or just use whatever we set. 

If there isn't a need for admins to configure the key, and since we are not using the service environment in the right way, why not remove it altogether and keep it simple. i.e.:
- Hardcode the test key in the Makefile.
- And let that be overriden with an env var from CI pipelines. 
So now there are no keys in the codebase. And there is no need to check prod or test env. Just use whatever is passed from LDFLAGS.

Reply:
@agnivade is this the correct way to do if admins wants to set their own keys?

Reply:
Indeed it is.

Reply:
As discussed offline, yes, this works but it will store the default prod key in the config which we could avoid if we override at the time of returning the key in the api. We can leave the config only if the admin decides to configure their own key